Makefile.PL

```diff
@@ -5,7 +5,7 @@ use inc::Module::Install;
 name 'Zonemaster-GUI';
 abstract 'Web frontend for Zonemaster';
 author 'Michal TOMA <toma@nic.fr>';
-license 'perl';
+license 'bsd';
```

Comment:
Shouldn't it rather be 'https://opensource.org/licenses/BSD-2-Clause'?

Reply:
No, this field is a token, not a url.
http://search.cpan.org/~ether/Module-Install-1.18/lib/Module/Install.pod#license